pkg/plugins/golang/v4/scaffolds/internal/templates/makefile.go

```diff
@@ -119,6 +119,8 @@ help: ## Display this help.
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	# TODO: When controller-tools supports feature gates (issue #1238), add:
```

Comment:
```suggestion
```

I do not think that we should change the default scaffolding, adding the TODO:
But we definilly would need to document